src/language/runtime/values.ts

```diff
@@ -105,11 +105,12 @@ export interface MethodVal {
     body: Stmt<StmtKind.ReturnCommand>[];
 }
 
-export interface BuiltinClassVal {
+export interface BuiltinClassVal<C> {
     type: ValueAlias.Class;
     name: string;
     internal: true;
     prototype: ClassPrototype;
+    internalConstructor: ((args: RuntimeVal[]) => C) | null;
```

Comment:
```suggestion
    internalConstructor?: undefined | (args: RuntimeVal[]) => C;
```

Ich frag mich, ob der Methode auch mehr Dinge, wie das class proxy object mitgegeben werden sollten.
Unten noch `robotCls` und `worldClass` anpassen!

Reply:
Wie meinst du das? Also `internalConstructor?: undefined | (proxy: ProxyObjectVal<C>, args: RuntimeVal[]) => C`? Weiß nicht genau, welchen Nutzen das hätte. 

Danke fürs reviewen auf jeden Fall!